services/QuillLMS/spec/models/unit_activity_spec.rb

```diff
@@ -140,6 +142,62 @@
     end
   end
 
+  describe 'adjust_due_date_for_timezone' do
+    it 'adjusts the submitted time to the utc equivalent of that time in the timezone of the teacher' do
+      tz_string = 'America/New_York'
+      teacher.update(time_zone: tz_string)
+      due_date = Time.now.utc - 1.hour
+      unit_activity.due_date = due_date
+      unit_activity.adjust_due_date_for_timezone
+      expect(unit_activity.due_date).to eq(due_date + 4.hours)
```

Comment:
I'm not sure this is fixed yet.

Reply:
@brendanshean it was fixed in [this commit](https://github.com/empirical-org/Empirical-Core/pull/9788/commits/391ebba9c8ef69d1b56f1230bf3069d446967c33) (I updated it to use a calculated offset rather than a constant).

Reply:
Perfect.  For some reason the UI was still linking me to the original `4.hours` code